fastos/src/tests/thread_test_base.hpp

```diff
@@ -101,7 +101,7 @@ void ThreadTestBase::Run (FastOS_ThreadInterface *thread, void *arg)
 
          int sleepOn = (INCREASE_NUMBER_AMOUNT/2) * 321/10000;
          for (int i=0; i<(INCREASE_NUMBER_AMOUNT/2); i++) {
-            number = number + 2;
+            number += 2;
```

Comment:
Does this test try to check any thread safety issues by itself? If so, should this be a relaxed `fetch_add` to avoid introducing implicit barriers that may hide other things?

Reply:
might be, relaxing it